### ISSUE TITLE
Ignore php blocks

### ISSUE THREAD

**Issue by senor-developer** [praveenvijayan/grunt-html-validation#60]
Is there any way to tell the validator to skip over php tags? The project I'm currently working on relies heavily on php includes for factorization, but the files are otherwise 99% html. 
The validator runs flawless on php files, I just need it to ignore whatever is between <?php and ?>

Any ideas?

**Comment by BOOMER74** [comment]
Ignore is not good idea. Need to try execute `php` and validate output html.

**Comment by senor-developer** [comment]
Thanks for the suggestion, I got it to work using grunt-php2html to parse all php files into my .tmp folder and run htmlhint in there. 
The only drawback is that (obviously) line numbers are off as the line numbers in my IDE no longer match the parsed html files in .tmp. 

Can't think of any workaround right now, but any further suggestions would be much appreciated :)

**Comment by paulbhartzog** [comment]
Yes, is there a way to do html validation with grunt on php files other than php2html?
